behave/features/diagnosis.feature

```diff
@@ -21,13 +21,29 @@ Scenario: Given an INSCOPE decision when selecting debt and court proceedings
     And I get an "INSCOPE" decision
     And select 'Create financial assessment'
     Then I am taken to the Finances tab with the ‘Details’ sub-tab preselected
+    #This is the icon in the top RH corner
     When I select 'Assign Alternative Help'
```

Comment:
What do you of changing the step to include it's for an icon?

Reply:
good idea, will do.